Improvement/zenko 886 update nodejs

rahulreddy opened the pull request:
1. improvement: move metrics tests to functional
2. improvement: run tests in eve with nodejs 8
3. improvement: update test to adjust to nodejs 8
4. improvement: update package-lock version

rahulreddy committed:
improvement: move metrics tests to functional

rahulreddy committed:
improvement: run tests in eve with nodejs 8

bert-e commented:
# Hello rahulreddy,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get
information on this process.





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:



* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rahulreddy

* @RachedBenMustapha

* @bennettbuchanan

rahulreddy committed:
improvement: update test to adjust to nodejs 8

Buffer.from no longer throws errors if most of the string contains a valid hex.
Since the test is testing if an error is thrown for invalid hex, the test has been
updated to do the same.

rahulreddy commented:
@bert-e wait

rahulreddy committed:
improvement: update package-lock version

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.0`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of Release Engineering will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of Release Engineering now.

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.0`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`



Please check the status of the associated issue ZENKO-886.

Goodbye rahulreddy.
